pandas/core/internals/managers.py

```diff
@@ -1749,6 +1749,40 @@ def as_array(
 
         return arr.transpose()
 
+    def as_array_masked(self) -> tuple[np.ndarray, np.ndarray]:
+        """
+        Convert the blockmanager data into an numpy array.
+
+        Returns
+        -------
+        arr : ndarray
+        """
+        # # TODO(CoW) handle case where resulting array is a view
```

Comment:
@jorisvandenbossche - I don't understand this TODO, can you give more details?

Reply:
Hmm, I suppose that with the currently implementation, the resulting arrays are never views but always copies? (since `result_data` and `result_mask` are newly created arrays (and then the values are copied in) 
In that case I think you can just remove the comment

Reply:
Ah, that was just copied from the `as_array` implementation above, then can certainly be removed. Sorry for the confusion!